Source/SwipeAnimator.swift

```diff
@@ -0,0 +1,115 @@
+//
+//  SwipeCellAnimator.swift
+//  SwipeCellKit
+//
+//  Created by Daniel Carmo on 2017-03-22.
+//
+//
+
+import Foundation
```

Comment:
I like the direction you're going with this last update.

It would even better if this protocol matched the UIViewPropertyAnimator initializer and methods that we're currently using.  For the `UIViewPropertyCellAnimator`, it would be essentially passthrough.  For `UIViewCellAnimator`, it would have instance variables like you currently do, and we'd hold onto the animation closure via `addAnimations` and other properties, rather than the cell instance.